tests/integration/constraints/test_tabular.py

```diff
@@ -8,15 +8,21 @@
 from sdv.tabular import GaussianCopula
 
 
+def f1(_, x):
+    return pd.Series([True if x_i > 0 else False for x_i in x['col']])
+
+
+def f3(_, x):
+    return pd.DataFrame({'col': x['col'] ** .5})
+
+
 def test_create_custom_constraint():
     """Test the ``create_custom_constraint`` method end to end."""
     # Setup
-    custom_constraint = create_custom_constraint(
-        lambda _, x: pd.Series([True if x_i > 0 else False for x_i in x['col']]),
-        lambda _, x: pd.DataFrame({'col': x['col'] ** 2}),
-        lambda _, x: pd.DataFrame({'col': x['col'] ** .5})
-    )('col')
+    def f2(_, x):
```

Comment:
can we give these functions descriptive names so we can better understand the test? like `test_transform`, `test_reverse_transform` etc.